unstructured_ingest/v2/processes/connectors/onedrive.py

```diff
@@ -135,12 +135,12 @@ def drive_item_to_file_data(self, drive_item: "DriveItem") -> FileData:
         server_path = file_path + "/" + filename
         rel_path = server_path.replace(self.index_config.path, "").lstrip("/")
         date_modified_dt = (
-            parser.parse(drive_item.last_modified_datetime)
+            parser.parse(str(drive_item.last_modified_datetime))
             if drive_item.last_modified_datetime
             else None
         )
         date_created_at = (
-            parser.parse(drive_item.created_datetime) if drive_item.created_datetime else None
+            parser.parse(str(drive_item.created_datetime)) if drive_item.created_datetime else None
```

Comment:
This was raising `TypeError: Parser must be a string or character stream, not datetime` when trying to use `parser.parse(drive_item.last_modified_datetime)`